Basic-Car-Maintenance/Shared/Settings/Views/ContributorsListView.swift

```diff
@@ -14,7 +14,20 @@ struct ContributorsListView: View {
     var body: some View {
         List {
             if let contributors = viewModel.contributors, !contributors.isEmpty {
-                ForEach(contributors) { contributor in
+                let sortedContributors = contributors.sorted { (contributor1, contributor2) in
+                    
+                    // Sort by contributions (highest first), then by login name
+                    if contributor1.contributions > contributor2.contributions {
+                        return true
+                    } else if contributor1.contributions < contributor2.contributions {
+                        return false
+                    }
+                    
+                    // If contributions are equal, compare by login name
+                    return contributor1.login < contributor2.login
+                }
+                
+                ForEach(sortedContributors) { contributor in
```

Comment:
Can you move this into the `SettingsViewModel` instead, you could do something similar to `sortedEvents` that's in the `DashboardViewModel`

Using a computed property